phase_shifter_ds/phase_shifter.py

```diff
@@ -1,17 +1,25 @@
 """This is PhaseShifter device class based on the facadedevice library"""
 
-from facadedevice import Facade, proxy_attribute, logical_attribute, local_attribute
+from facadedevice import Facade, proxy_attribute, logical_attribute
 from tango import AttrWriteType, DevState
 from tango.server import device_property
 import csv
 
 
 class PhaseShifter(Facade):
-
+    """
+    This PhaseShifter class is based on the facadedevice library.
+    Its main purpose is to change received voltage into degrees accordingly to
+    the data provided in external file. During initialization, data from this file
+    is imported into two lists: one for voltage and one for degrees. In order to
+    do it user has to specify ConfigurationPath (path to the .csv file).
```

Comment:
Items such as `facadedevice`, `ConfigurationPath` or `.csv` should be marked as code (surrounded with backticks), since they're not correct from the language point of view. The generated documentation will also look nicer.